nncf/experimental/quantization/compression_builder.py

```diff
@@ -66,6 +67,7 @@ def _get_prepared_model_for_compression(self, model: TModel, backend: BackendTyp
 
         return None
 
+    @tracked_function(NNCF_ONNX_CATEGORY, [CompressionStartedFromBuilder(argname="self"), ])
```

Comment:
We are going to remove the `CompressionBuilder` class in future PRs. So I think it is better to decorate the `quantize_impl()` methods directly.

Reply:
Added the decorator to the `quantize_impl` as well.